### DISCUSSION TITLE
A tax system could be good.

### DISCUSSION THREAD

**Discussion by th3lucas** [Zrips/CMI#7203] [Feature Requests]
### Discussed in https://github.com/Zrips/CMI/discussions/5163

<div type='discussions-op-text'>

<sup>Originally posted by **Mcdikmen** February 24, 2021</sup>
Hello,

I want to suggest u a tax system. It's simple and very effective for some server types i think. Cuz i dont want my players reaching too many balances. If i am tax my player balances every week like a percentage (and offline players too), it could be good. Maybe some players may receive more taxes. Like:

0-10.000 K balances, recieving %5
10.000-50.000 K balances, recieving %10
50.000+ balances, recieving %20

Hope you like it. Thanks for reading.</div>

**Comment by mrfloris** [comment]
Duplicate of https://github.com/Zrips/CMI/discussions/5163
